src/components/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom'
+import '../App.css';
+
+class Navbar extends React.Component {
+  render (){
+    return (
+      <div>
+          <nav className="Navbar">
+            <Link to='/'>
+                <img className='logo-pic' src='https://www.freeiconspng.com/uploads/clapperboard-png-1.png' alt='logo' />
+            </Link>
+            <Link to='/location'>Locations</Link>
+            <Link to='/movies'>Movies</Link>
+            <Link to='/people'>People</Link>
+          </nav>
+      </div>
+    );
```

Comment:
```suggestion
    return (
          <nav className="Navbar">
            <Link to='/'>
                <img className='logo-pic' src='https://www.freeiconspng.com/uploads/clapperboard-png-1.png' alt='logo' />
            </Link>
            <Link to='/locations'>Locations</Link>
            <Link to='/movies'>Movies</Link>
            <Link to='/people'>People</Link>
          </nav>
    );
```
I have two things to say about your navbar:
1) You don't need to wrap everything in a `nav` element AND a `div`. Just a `nav` is enough :) 
2) This second point is more important. It's actually keeping your code from working correctly. There's a typo in your `Link` for Locations. It doesn't **perfectly match** your `Route` path for the `Locations` component. That's why this `Link` doesn't work properly when you click on it.